Docs/source/Software/BSP.rst

```diff
@@ -38,4 +38,20 @@ Fans BSP: Manthan Upadhyaya
 
 The Fans driver is used to control the speed of the fans depending on the temperature of the Battery
 Pack. It allows for as many speeds as needed and can change the speed of individual fans. It supports
-four fans and uses PWM to change the speed and hopefully save energy. 
\ No newline at end of file
+four fans and uses PWM to change the speed and hopefully save energy. 
+
+CAN BSP: Chase Block & Sijin Woo
+================================
+
+Purpose
+    The purpose of this code is to send and recieve CAN messages. It also contains a queue for 
+    messages that are recieved on the CAN bus that has a depth of 10. This queue cannot be accessed
+    outside the file.
```

Comment:
I think it would make more sense to not put anything about the queue in here. I would instead put it in additional considerations